packages/client/wallets/smart-wallet/README.md

```diff
@@ -0,0 +1,261 @@
+# Smart Wallet SDK
+
+A client-side SDK for interacting with Crossmint Smart Wallets. This SDK enables developers to create and manage smart contract wallets that support both traditional keypair-based signing and passkey authentication.
+
+## Installation
+
+```sh
+npm install @crossmint/client-sdk-smart-wallet
+# or
+pnpm add @crossmint/client-sdk-smart-wallet
+```
+
+## Authentication
+
+This SDK uses JSON Web Tokens (JWT) to authenticate your users. JWT is generated by the auth server and used to authorize users to deploy and operate their wallets.
+
+For detailed instructions on how to obtain a JWT token, please refer to [User Auth](https://docs.crossmint.com/wallets/quickstarts/EVM/non-custodial-wallets/evm-non-custodial-client-side#4-set-up-user-auth).
+
+## Quick Start
+
+```typescript
+import { SmartWalletSDK } from "@crossmint/client-sdk-smart-wallet";
+import { privateKeyToAccount } from "viem";
+
+const sdk = SmartWalletSDK.init({ clientApiKey: YOUR_API_KEY });
+const account = privateKeyToAccount(privateKey);
+
+const wallet = await sdk.getOrCreateWallet(
+  {
+    jwt: USER_TOKEN,
```

Comment:
Did the previous version also expect a JWT as the first param?

Reply:
yes, exact same API